interop/cats/src/main/scala/caliban/interop/cats/FromEffect.scala

```diff
@@ -0,0 +1,49 @@
+package caliban.interop.cats
+
+import cats.~>
+import cats.effect.Async
+import cats.effect.std.Dispatcher
+import zio.{ RIO, Runtime }
+
+trait ToEffect[F[_], R] {
```

Comment:
The file is named `FromEffect.scala`, maybe rename of split this into 2 files?

Reply:
I split this into 2 files and updated docs.